feat: Update operator docs (#858)
vegaprotocol/documentation: versioned_docs/version-v0.73/node-operators/get-started/setup-datanode.md

The default value of the `huge_pages` config is `try`. Setting it to `off` usually reduces the RAM usage, however, it increases the CPU usage on the machine.

#### Max locks per transaction

New Value:

```conf
max_locks_per_transaction = 256
```

> This value may change in future to bigger number. It depends on the traffic on the network. See [the official docs](https://postgresqlco.nf/doc/en/param/max_locks_per_transaction/) for reference.

Reviewer: ```suggestion
This value may change in the future to a bigger number, depending on the network traffic. See the [Postgres docs ↗](https://postgresqlco.nf/doc/en/param/max_locks_per_transaction/) for reference.
```

> :::note

Reviewer: ```suggestion
:::note Error state
```

> When the value is too low, you may end with the following error: `ERROR: out of shared memory (SQLSTATE 53200)`.

Reviewer: ```suggestion
When the value is too low, you may see the following error: `ERROR: out of shared memory (SQLSTATE 53200)`.
```

```
4. Now start the non validator node and confirm that both apps are running and you can see the block height increasing on both. 

:::warning

> We found a bug in the vega network that crashed the mainnet network at block `26439343`. This makes replaying chain from block 0 more complicated. 
> 
> ```
> cannot unregister order with potential sell + size changes < 0
> ```
> 
> When your node failed with the above error message the procedure is following:
> 
> 1. Rollback tendermint block: `vega tm rollback --home <tendermint_home>
> 2. Download the [v0.73.6-patch.1](https://github.com/vegaprotocol/vega/releases/tag/v0.73.6-patch.1) binary
> 3. Start your node with downloaded binary from block `26439116`, e.g: with the following flag: `--snapshot.load-from-block-height 26439116`
> 4. The binary will apply some fixes to the broken transactions, but  the node will fail with the following error: `panic: cannot unregister order with potential sell + size change < 0`.
> 5. Reply steps 1-3, but use the [v0.73.6-patch.2](https://github.com/vegaprotocol/vega/releases/tag/v0.73.6-patch.2) binary.
> 
> **It is an important to let binary fail with the `v0.73.6-patch.1` binary, or you won't be able to move your node forward!.**
> 
> After v0.73.6-patch.2 your node will normally continue replaying.

Reviewer: ```suggestion
A bug on the Vega network that crashed the mainnet network at block `26439343`. This makes replaying chain from block 0 more complicated. 

If your node fails with this error message, follow the procedure described below: `cannot unregister order with potential sell + size changes < 0`

1. Roll back tendermint block: `vega tm rollback --home <tendermint_home>`
2. Download the [v0.73.6-patch.1 ↗](https://github.com/vegaprotocol/vega/releases/tag/v0.73.6-patch.1) binary
3. Start your node with downloaded binary from block `26439116`, e.g: with the following flag: `--snapshot.load-from-block-height 26439116`
4. The binary will apply some fixes to the broken transactions, but the node will fail with the following error: `panic: cannot unregister order with potential sell + size change < 0`.
5. Reply steps 1-3, but use the [v0.73.6-patch.2 ↗](https://github.com/vegaprotocol/vega/releases/tag/v0.73.6-patch.2) binary.

**It is essential to let the binary fail with the `v0.73.6-patch.1` binary, or you won't be able to move your node forward!**

After `v0.73.6-patch.2` your node will continue replaying normally.
```
